testsuite/tests/lib-array/test_iarray_typeopt.ml

```diff
@@ -5,6 +5,10 @@
    expect;
 *)
 
+(* CR dkalinichenko: this test generates slightly different code
```

Comment:
Is the difference just that the `name/number` identifiers change? If it's more than that it seems concerning.

Reply:
Yeah, that's right.